app/models/sidebar_section.rb

```diff
@@ -53,6 +55,11 @@ def reset_community!
     end
   end
 
+  def preload(associations)
```

Comment:
Kind of strange that Rails doesn't have an API for preloading associations for a single object.

Reply:
`includes` is a query method API. In this case, we're dealing with an ActiveRecord object which isn't a query object.